addons/l10n_ae/__manifest__.py

```diff
@@ -4,6 +4,7 @@
     'name': 'U.A.E. - Accounting',
```

Comment:
```suggestion
    'name': United Arab Emirates - Accounting',
```
In order to be consistent with other localization modules, it would make more sense to use the full country name, like it's done in the US for example:
https://github.com/odoo/odoo/blob/6db2847b476a21d6a96baddecb8231e47157ae0e/addons/l10n_us/__manifest__.py#L5-L8
(same for `l10n_ae_pos`).